docs/source/INSTRUCTORS.md

```diff
@@ -0,0 +1,91 @@
+
+### How is an instructor associated with students?
+
+Students and instructors will be matched based on overlapping research interests and skill sets in a pod to meet regularly.
+
+
+### How is an instructor expected to communicate with the students?
+
+The communication across the day is run through an international Discord server. The instructors will encourage students to use the module/project-related channels to raise their questions and get feedback from their peers.
+
+Students will also have a chance to raise their questions during the Q&A hours when the instructor is expected to meet in person with the members of the pod they are assigned to.
+
+
+### Does an instructor have to answer the questions from students from other hubs?
+
+They are expected to but do not have to. Each hub will primarily be responsible for answering and helping its students. However, resolving some queries raised in the Discord server might address your expertise, or you might already know the answer. Then it would help the school community to resolve the issues quickly. 
+
+### How will an instructor follow the progress of a student?  
+
+For tracking the progress with students' completion of the module assignments, the local hub organizers that you are affiliated with will provide a means, mostly a shared sheet with the students' names and modules. So you will issue whether a student has completed an assignment based on their individual reporting.
+
+For the project improvements and assignments, meeting with the students 2 times a day will give you likely observations regarding how a student is progressing with their needs and executions.
+
+### What am I expected to do during a clinic/Q&A call?
+
+During the Q&A hours in weeks 1 and 2, you are expected to catch up with the students regarding their module assignments and queries. 
+
+
+During the clinic hours in weeks 3 and 4, you will track the student's progress with their project execution and share your feedback if they have any specific queries.
+
+Please do not forget you are not expected to provide definite help or the correct answer. You are there to give feedback and point to the possible resources that might help them. 
+
+
+### What should I do if a student is not progressing with their project?
+
+Sometimes some students are stuck with an idea, choosing the right tool, or finding the data for their project. In some cases, they might also be hesitant to raise their questions. 
+
+The best way to help those students would be to ask what aspect of their project is preventing them from progressing and give feedback to them to narrow down their project scope to facilitate execution.
+
+Please remember you are there to provide means for students to progress rather than tackling the question together. You are not expected to boggle down the code or execute a task for them, or not to micro-manage their progress but to give them insight and guidance regarding how they can move forward in a possible way. 
```

Comment:
this is a bit repetitive of line 65. Suggest editing both together in one section, rephrasing or omitting one part